bokehjs/test/integration/glyphs/markers.ts

```diff
@@ -97,4 +90,25 @@ describe("Marker glyph", () => {
 
     await display(row([p0, p1, p2]))
   })
+
+  it("should support multiple marker types in scatter", async () => {
+    const marker_type = [
+      "asterisk", "circle", "square_dot", "triangle_pin", "star",
+      "x", "diamond_cross", "star", "circle_cross", "hex_dot"]
```

Comment:
Preferably use a trailing comma and move `]` to a new line. It's easier to modify an array this way.